src/lat/lattice-functions.cc

```diff
@@ -1646,4 +1649,110 @@ void ComposeCompactLatticeDeterministic(
   fst::Connect(composed_clat);
 }
 
+
+void ComputeAcousticScoresMap(
```

Comment:
if it turns out you don't need this code after looking into the composition, you can remove it.